Makefile

```diff
@@ -1,7 +1,9 @@
 -include ../../config.mk
 -include ../../tools/kconfig/.config
 
-CC:=$(CROSS_COMPILE)gcc
+OS:=$(shell uname)
+
+CC:=$(CROSS_COMPILE)clang
```

Comment:
Created a new pull request with just this part: https://github.com/tass-belgium/picotcp/pull/463